About.html

```diff
@@ -0,0 +1,87 @@
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+                <li><a class="menu-item" href="#logo">logo</a></li>
+                <li><a class="menu-item" href="#vision">Vision</a></li>
+            </ul>    
+        </nav>
+    </header>    
+        <section class="grtittle">
+            <h1>Kamulete training center(Rwanda)</h1>
+            <hr class="linebar">
+            <div class="centerpar">
+                <p>kamulete trainig center for all people around the world,<br> we invite you to join our training center.
+                    kamulete training center was created in 2023 by kamulete bertin who is the CEO form Rwanda,<br> he found that to train people by knwowing many languages is very helps full and develops creativness in someone.
+                    with that this trainig center give you a diploma after complited the month acording to your program choosen,<br> that diploma is valid in each country around the word.
+                    we have more than 20 teachers by which we have collaboration and they helps us to give some online classes.
+                </p>
+            </div>
+          <div class="contactclass">
+            <p class="contactclass1">please contact us via email for more information.    </p>
+            <p class="contactclass2">email: Kamuletetrainingcenter988@gmail.com<br></p>
+            <p class="contactclass3">@250 kigali , palace (Rwanda)</p>
+          </div>
+        </section>
+        <section id="logo" class="classlog">
+            <h2>kamulete-training-center-logo</h2>
+            <hr class="linebar">
+            <p>the logo of kamulete traning center logo was decided from a competition from 12 january to 30january.
+                After finishing the competition voting and selection committe's judjjing process,<br> the logo from 20 years oldswon the prize-winning-logo for kamulete training center as follow
+            </p>
+            <img src="img-folder/logo-image.png" alt="">
+        </section>
+        <section id="vision" class="vissonsee">
+            <h2>see our vision and goals </h2>
+            <hr class="linebar">
+            <p>see how we working with our students</p>
+           <div>
+           <p class="section-red">Different community</p>
+            <p class="section-red1">Different community</p>
+            </div>
+        </section>
+        <section class="partners">
+            <h3>partners</h3>
+            <hr class="linebar">
+            <ul>
+                <li>kivumotor</li>
+                <li>KinMarch</li>
+                <li>Vinkingboy</li>
+            </ul>
+        </section>
+        <footer>
+            <img src="img-folder/logo-image---00.png" alt="">
+            <p>@ 2023 ,Authorized by Rwanda Education Boards</p>
+            <p> <p>© Terms and Conditions apply.</p> </p>
+        </footer>
```

Comment:
[OPTIONAL] the footer in mobile size screens is not aligned the same way as requested kindly fix it 

![footer](https://user-images.githubusercontent.com/59707859/225984115-b5927d24-cf27-4ca1-81cf-83c679843252.PNG)

![Uploading your footer.PNG…]()